src/arm-elf-adac/Makefile

```diff
@@ -0,0 +1,30 @@
+## This file is part of the adac-hybrid-arm distribution
+## Copyright (C) 2020, European Space Agency
+##
+## This program is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation, version 3.
+##
+## This program is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+TOP_DIR=$(shell realpath ../../)
```

Comment:
Is this used?

Reply:
Yes, in BIN_DIR, which is one line below... and which is used in the formulas.
BIN_DIR is not assembled directly, as argument to realpath must exist...